data/gmh/Junk_1905_WillehalmVonOrleans/stemma.gv

```diff
@@ -0,0 +1,65 @@
+digraph {
+arch[label="", color="grey"]

```

Comment:
Great ! Given the shape of the graph, I would be tempted to add
```gv
original[label="", color="grey"];
original->arch;
```
at the top. What do you think ?

Reply:
Absolutely!